src/StreamJsonRpc/Reflection/MessageFormatterDuplexPipeTracker.cs

```diff
@@ -92,6 +92,7 @@ public MessageFormatterDuplexPipeTracker()
         /// When the response is received, a call should always be made to <see cref="OnResponseReceived(RequestId, bool)"/>.
         /// </remarks>
         /// <exception cref="NotSupportedException">Thrown if no <see cref="MultiplexingStream"/> was provided to the constructor.</exception>
+        [return: NotNullIfNotNull("token")]
```

Comment:
Shouldn't this be duplexPipe? Similar for GetToken below as well

Reply:
Yes. I'll fix that. That seems like something C# ought to have an analyzer to catch. @sharwell, are you aware of anything on the backlog for this?

Reply:
I believe it's item 11 in #36073